src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php

```diff
@@ -50,8 +50,11 @@ class CachingType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('use_cache', SwitchType::class)
+            ->add('use_cache', SwitchType::class, [
```

Comment:
you should disable the translation of the form component: otherwise the translation will be done twice => https://symfony.com/doc/3.4/reference/forms/types/form.html#translation-domain

you can control the behavior in the Translations profiler (this is something you can actually almost "unit test").

Reply:
Good tip 👍